src/themes/teams/components/Chat/chatItemStyles.ts

```diff
@@ -1,25 +1,76 @@
 import { ICSSInJSStyle, ComponentSlotStylesInput } from '../../../types'
 import { ChatItemVariables } from './chatItemVariables'
 import { ChatItemProps } from '../../../../components/Chat/ChatItem'
+import { pxToRem } from '../../../../lib'
+import ChatMessage from '../../../../components/Chat/ChatMessage'
+
+const chatMessageClassname = `& .${ChatMessage.className}`
+const chatMessageAuthorClassname = `& .${ChatMessage.className}__author`
+const chatMessageTimestampClassname = `& .${ChatMessage.className}__timestamp`
```

Comment:
Yep, will rename those, although still not sure how to export this slot's classnames from the components... Should we add authorClassName, timestampClassName as static fields in the ChatMesage?! Maybe just a constants exported from the ChatMessage (although it will be not consistent as other class names defined).

Reply:
Would like to avoid the name selectors as it indicates that we will have the class selector `.` at the end of the string. We may add something like ChatMessage.classes.author, but if we decide in future to have some different selectors (other then the classes, then maybe your suggestion would make more sense for the user... Just like to mention that if we decide with the selectors, would like to specify the authors and timestamp there as `.ui-chat__item__author` and `.ui-chat__item__timestamp` respectively. @kuzhelov @Bugaa92 @layershifter  what is your opinion on this?

Reply:
Will add a static field called cssSelectors, and will specify the selectors for the author and timestamp there.

Reply:
My opinion: we call `timestamp` and `author` - slots, so may be it can be `slotClassNames` and add it as a static field to matching component

Reply:
Later we can introduce a test that asserts that all keys that are presented in styles have matching `slotClassName`

Reply:
Maybe `slotCssSelectors` will be better name, as in future we may have more complex selectors that are not necessarily classNames. And defining the selector will allow the user to always use them consistently without adding the necessary selector for className for example.. What do you think?

Reply:
1. As user I want to have CSS class names because I should be able to build custom selectors:
```js
`.${ChatItem.slotCssSelectors.author} + .${Another.slotCssSelectors.slot}`
```

2. We have `className` property on each component, it's strange to have `selectors`